build/cmake/DefineOptions.cmake

```diff
@@ -116,11 +116,12 @@ CMAKE_DEPENDENT_OPTION(BUILD_NODEJS "Build NodeJS library" ON
                        "BUILD_LIBRARIES;WITH_NODEJS" OFF)
 
 # Python
-option(WITH_PYTHON "Build Python Thrift library" ON)
+option(WITH_PY3 "Build Python Thrift library" ON)
+set(Python_ADDITIONAL_VERSIONS 3.6 3.7 3.8 3.9 3.10)
```

Comment:
3.6 is already unsupported according to https://devguide.python.org/versions/, so we only need to support 3.7+.

Reply:
removed in https://github.com/apache/thrift/pull/2588/commits/67ac427c2ce81a594f32e3d9b5871e760674502d